README.md

```diff
@@ -1,42 +1,39 @@
 
-With the development of large language models, their underlying network structure, the Transformer, is being extensively studied. Researching the Transformer structure helps us enhance our understanding of this "black box" and improve model interpretability. Recently, there has been an increasing body of work suggesting that the model contains two distinct partitions: attention mechanisms used for behavior, inference, and analysis, and feed-forward networks (FFN) for knowledge storage. The former is crucial for revealing the functional capabilities of the model, leading to a series of studies exploring various functions within attention mechanisms, which we have termed **Attention Head Mining**.
-
-Ordered by publication date:
-
-* ![](https://img.shields.io/badge/Iteration%20Head-blue) Iteration Head: A Mechanistic Study of Chain-of-Thought. arXiv 2024. [[Paper](https://arxiv.org/abs/2406.02128)]
-
-* ![](https://img.shields.io/badge/Mover%20Head-blue) ![](https://img.shields.io/badge/Relation%20Head-blue) Knowledge Circuits in Pretrained Transformers. arXiv 2024. [[Paper](https://arxiv.org/abs/2405.17969)] [[Code](https://github.com/zjunlp/KnowledgeCircuits)]
-
-* ![](https://img.shields.io/badge/Induction%20Head-blue) Induction Heads as a Primary Mechanism for Pattern Matching in In-context Learning. OpenReview 2024. [[Paper](https://openreview.net/forum?id=np6hrTv7aW)]
-
-* ![](https://img.shields.io/badge/Retrieval%20Head-blue) Retrieval Head Mechanistically Explains Long-Context Factuality. arXiv 2024. [[Paper](https://arxiv.org/abs/2404.15574)] [[Code](https://github.com/nightdessert/Retrieval_Head)]
-
-* ![](https://img.shields.io/badge/Truthfulness%20Head-blue) Non-Linear Inference Time Intervention: Improving LLM Truthfulness. arXiv 2024. [[Paper](https://arxiv.org/abs/2403.18680)] [[Code](https://github.com/Samsung/NL-ITI)]
-
-* ![](https://img.shields.io/badge/Induction%20Head-blue) Identifying Semantic Induction Heads to Understand In-Context Learning. arXiv 2024. [[Paper](https://arxiv.org/abs/2402.13055v1)]
+</div>
 
-* ![](https://img.shields.io/badge/Induction%20Head-blue) The Evolution of Statistical Induction Heads: In-Context Learning Markov Chains. arXiv 2024. [[Paper](https://arxiv.org/abs/2402.11004)]
+Welcome to **Awesome-Attention-Heads**! This is the platform to get the latest research on Attention Heads. We hope to provide complete and clear cutting-edge informations for researchers studying LLM interpretability and LLM hallucination.
 
-* ![](https://img.shields.io/badge/Mover%20Head-blue) ![](https://img.shields.io/badge/Relation%20Head-blue) Summing Up the Facts: Additive Mechanisms Behind Factual Recall in LLMs. arXiv 2024. [[Paper](https://www.arxiv.org/abs/2402.07321)]
+test
```

Comment:
Check this

Reply:
Sorry for the mistake. I've already remove it.